src/plots/cartesian/axes.js

```diff
@@ -129,12 +173,13 @@ axes.coerceRef = function(containerIn, containerOut, gd, attr, dflt, extraOption
  * - for other types: coerce them to numbers
  */
 axes.coercePosition = function(containerOut, gd, coerce, axRef, attr, dflt) {
-    var cleanPos, pos;
-
-    if(axRef === 'paper' || axRef === 'pixel') {
+    var cleanPos, pos, axRefType;
+    axRefType = axes.getRefType(axRef);
```

Comment:
```suggestion
    var cleanPos, pos;
    var axRefType = axes.getRefType(axRef);
```
Minor style point - list uninitialized vars together, but if you're going to initialize a var just declare and initialize it in one statement on its own line.